XamarinCommunityToolkit/Effects/EffectIds.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Xamarin.Forms;
+
+[assembly: ResolutionGroupName("Xamarin.CommunityToolkit.Effects")]
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public sealed class EffectIds
```

Comment:
Should we make it internal?
I don't think we should expose it for XCT users

Reply:
`internal`? even better! I'll update it now

Reply:
Interestingly the stylecop shouts at me here

![Stylecop shouting at me](https://user-images.githubusercontent.com/33064621/97421882-ed7eb600-1904-11eb-8872-c98a783c370d.png)

The suggested code change is:
`sealed class EffectIds`

Could there be any issues with this not being explicitly declared as `internal`? Without an accessibility modifier it defaults to `private` AFAIK. According to the [Access Modifiers documentation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers):
> internal: The type or member can be accessed by any code in the same assembly, but not from another assembly.
> private: The type or member can be accessed only by code in the same class or struct.

So `internal` is the correct modifier to use here. Do I make the change with the warning, do I suppress the warning? What are your thoughts?

Reply:
For classes **internal** is default :) so, yes, just leave "sealed class"

Reply:
done